tox.ini

```diff
@@ -60,8 +60,9 @@ deps = {[testenv]deps}
 changedir=test/unit
 commands =
     bash -c 'cd ../../ && ./setup.py develop'
-    pytest --no-cov-on-fail --cov=kiwi \
-        --cov-report=term-missing --cov-fail-under=100 --cov-config .coveragerc {posargs}
+    pytest --doctest-modules --no-cov-on-fail --cov=kiwi \
+        --cov-report=term-missing --cov-fail-under=100 \
+        --cov-config .coveragerc {posargs}
```

Comment:
Changes in tox.ini do not belong to this pull request and while I agree with the ones done here they should be done in an extra pull request to avoid confusion

Reply:
Will drop them then.